taskcluster/utils.py

```diff
@@ -276,6 +317,23 @@ def authenticate(description=None):
     The description will be shown on login.taskcluster.net, if not provided
     a default message with script path will be displayed.
   """
+
+  # Let's first offer to read credentials from a configuration file then
+  # the environment
```

Comment:
We can also have an option to save the credentials to disk... so we can skip the step where we ask the user...